homeassistant/components/progettihwsw/config_flow.py

```diff
@@ -77,17 +80,19 @@ async def async_step_user(self, user_input=None):
         """Handle the initial step."""
         errors = {}
         if user_input is not None:
+
             try:
                 info = await validate_input(self.hass, user_input)
                 user_input.update(info)
```

Comment:
Please only wrap the line that can raise in the try... except block. Move the other lines to the `else:` block.

Reply:
Solved!